mkosi

```diff
@@ -1953,17 +1953,26 @@ def install_centos(args: CommandLineArguments, root: str, do_run_build_script: b
 
     packages = ['centos-release']
     packages += args.packages or []
+
     if args.bootable:
-        packages += ["kernel", "systemd-udev", "binutils"]
+        packages += ["kernel", "systemd", "binutils"]
```

Comment:
We should probably install systemd by default to align with the other distros. I know at least Arch and ubuntu install systemd by default. We want to unify this so that mkosi boot always works regardless if --bootable or not is specified (--bootable applies more to being able to boot via qemu).

Does centos-release not install systemd by default?